_data/l10n.yml

```
@@ -393,5 +406,12 @@ en:
         * **User community**: the number of unique authors;
 
         * **Longevity**: the age of the project. The ranges of every measure can be found in the vitality-ranges.yml file in [https://github.com/italia/developers-italia-backend](https://github.com/italia/developers-italia-backend)"
+    pa:
+      top_right_text: This page lists the software solutions owned by this public entity and released under an open license according to art. 69 of the Digital Administration Code.
```

```suggestion
      top_right_text: This page lists the software solutions owned by this public entity and released under an open license according to art. 69 of the Italian Digital Administration Code.
```
potrebbe aver senso aggiungere `Italian`?